code/game/turfs/open/floor.dm

```diff
@@ -235,8 +235,6 @@
 		if(STAGE_FIVE to INFINITY)
 			if(prob(70))
 				sheer = TRUE
-			else if(prob(50) && (/turf/open/space in baseturfs))
```

Comment:
Never hit, because the baseturf is always baseturf_bottom. @LemonInTheDark said to just remove.